packages/design-system/src/components/Alert/Alert.tsx

```diff
@@ -151,9 +151,9 @@ export const Alert: React.FC<AlertProps> = (props: AlertProps) => {
       {...alertProps}
     >
       {getIcon()}
-      <div className="ds-c-alert__body" id={headingId} ref={bodyRef}>
+      <div className="ds-c-alert__body" ref={bodyRef}>
         {heading ? (
-          <div className="ds-c-alert__header ds-c-alert__heading" ref={headingRef}>
+          <div id={headingId} className="ds-c-alert__header ds-c-alert__heading" ref={headingRef}>
```

Comment:
What happens if there is no heading?

Reply:
Oh, I see that the label goes to `heading ? headingId : a11yLabelId`

Reply:
That ternary gives me more reason to think the original placement of the `headingId` was incorrect.